docs/source/advanced/link-prediction.rst

```diff
@@ -49,44 +49,85 @@ GraphStorm provides two ways to compute link prediction scores: Dot Product and
     the ``tail_emb`` is the node embedding of the tail node and
     the ``relation_emb`` is the relation embedding of the specific edge type.
 
+.. _link_prediction_loss:
+
 Link Prediction Loss Functions
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-GraphStorm provides three options to compute training losses:
+GraphStorm provides four options to compute training losses:
 
 * **Cross Entropy Loss**: The cross entropy loss turns a link prediction task into a binary classification task. We treat positive edges as 1 and negative edges as 0. The loss of an edge ``e`` is as:
 
     .. math::
-        \begin{eqnarray}
-            loss = - y \cdot \log score + (1 - y) \cdot \log (1 - score)
-        \end{eqnarray}
+
+        loss = - y \cdot \log score + (1 - y) \cdot \log (1 - score)
 
     where ``y`` is 1 when ``e`` is a positive edge and 0 when it is a negative edge. ``score`` is the score value of ``e`` computed by the score function.
 
 * **Weighted Cross Entropy Loss**: The weighted cross entropy loss is similar to **Cross Entropy Loss** except that it allows users to set a weight for each positive edge. The loss function of an edge ``e`` is as:
 
     .. math::
-        \begin{eqnarray}
-            loss = - w\_e \left[ y \cdot \log score + (1 - y) \cdot \log (1 - score) \right]
-        \end{eqnarray}
+
+        loss = - w\_e \left[ y \cdot \log score + (1 - y) \cdot \log (1 - score) \right]
 
     where ``y`` is 1 when ``e`` is a positive edge and 0 when it is a negative edge. ``score`` is the score value of ``e`` computed by the score function, ``w_e`` is the weight of ``e`` and is defined as
 
     .. math::
-        \begin{eqnarray}
+
         w\_e = \left \{
         \begin{array}{lc}
             1,  & \text{ if } e \in G, \\
             0,  & \text{ if } e \notin G
         \end{array}
         \right.
-        \end{eqnarray}
 
     where ``G`` is the training graph.
 
+
+* **Adversarial Cross Entropy Loss**: The adversarial cross entropy loss turns a link prediction task into a binary classification task. We treat positive edges as 1 and negative edges as 0. In addition, adversarial cross-entropy loss adjusts the loss value of each negative sample based on its degree of difficulty. This is enabled by setting the ``adversarial_temperature`` config.
+
+    The loss of positive edges is as:
+
+    .. math::
+
+        loss_{pos} = - \log score
+
+    where ``score`` is the score value of the positive edges computed by the score function.
+
+    The loss of negative edges is as:
+
+    .. math::
+
+        loss_{neg} &= \log (1 - score)
+
```

Comment:
```suggestion
```

Reply:
Sphinx doc suggests to use empty line to separate two equations.